transactions/types.go

```diff
@@ -55,9 +64,17 @@ type SendTxArgs struct {
 	Input types.HexBytes `json:"input"`
 	Data  types.HexBytes `json:"data"`
 
-	// additional data
+	// additional data - version SendTxArgsVersion1
 	MultiTransactionID wallet_common.MultiTransactionIDType
 	Symbol             string
+	// additional data - version SendTxArgsVersion2
+	ValueOut           *hexutil.Big
+	FromChainID        uint64
+	ToChainID          uint64
+	FromTokenID        string
+	ToTokenID          string
+	ToContractAddress  types.Address
```

Comment:
it's not clear what this parameter's meaning is without looking at the code, could you add some comment?

Reply:
@dlipicar comment added.